app/src/main/java/cm/aptoide/pt/home/HomePresenter.java

```diff
@@ -535,6 +535,7 @@ private void handleEditorialCardClick() {
   }
 
   private void handleLoadMoreError() {
+    home.setLoadMoreError();
```

Comment:
I think this should be abstracted from the presenter. I would do this inside home itself, without having to come to the presenter.
Inside the loadNextHomeBundles i would check for the existence of an error and i would set it there.